.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules
```

Comment:
I'd suggest you move this to your .gitignore_global, it's nicer to have each person's OS specific stuff (.DS_Store) there